code/datums/explosion.dm

```diff
@@ -61,14 +61,14 @@ GLOBAL_LIST_EMPTY(explosions)
 		flash_range = min(GLOB.MAX_EX_FLASH_RANGE, flash_range)
 		flame_range = min(GLOB.MAX_EX_FLAME_RANGE, flame_range)
 
-	//DO NOT REMOVE THIS SLEEP, IT BREAKS THINGS
+	//DO NOT REMOVE THIS STOPLAG, IT BREAKS THINGS
```

Comment:
This one is cool for me to change up because I wrote this comment

Reply:
Note that I originally changed it to a stoplag and then back to a sleep because it slowed down explosion startups which I didn't approve of

Reply:
The formula stoplag uses is based off of a metric given by the mc, that metric was bugged because of a flipped operator making the mc think it byond resumed it late any time byond resumed it early, as well as a rounding error making that effect even worst.

Very likely under this pr we should see stoplag sleeping for longer then a tick much less often.